### ISSUE TITLE
The save button in the profile page should change to loading state when it is submitting for better UX

### ISSUE THREAD

**Issue by Sreejit-Sengupto** [RocketChat/Rocket.Chat#30317]
### Steps to reproduce:

1.  Go to Profile, make any changes for the save button to appear.
2.  Click the save Save button.
3.  Without a loading state, it is difficult for the user to understand whether the changes are being saved or any problem has occurred.

### Actual behavior: It gives no feedback to the users.

![image](https://github.com/RocketChat/Rocket.Chat/assets/69980616/1e85dccf-7ad1-4d28-be3c-5f07f25b2192)

### Server Setup Information:

- Version of Rocket.Chat Server: 
- Operating System: 
- Deployment Method: 
- Number of Running Instances: 
- DB Replicaset Oplog: 
- NodeJS Version: 
- MongoDB Version:

### Client Setup Information

- Desktop App or Browser Version:
- Operating System:

**Comment by AniruddhGuttikar** [comment]
will changing the opacity to 0.8 times the original value work?

**Comment by Sreejit-Sengupto** [comment]
> will changing the opacity to 0.8 times the original value work?

Yes it will. The user should get some feedback.

**Comment by AniruddhGuttikar** [comment]
oh okay, I'll try to fix that. Now trying to clone the repo locally (my first time cloning a repo)

**Comment by yashwardhan-jyani** [comment]
When you click on save profile button it does give feedback

See the top right corner
![Screenshot from 2023-09-08 00-24-13](https://github.com/RocketChat/Rocket.Chat/assets/100014271/bd5292c3-45ec-4e43-9bb1-310d56e8e349)

**Comment by Sreejit-Sengupto** [comment]
> When you click on save profile button it does give feedback
> 
> See the top right corner
> ![Screenshot from 2023-09-08 00-24-13](https://github.com/RocketChat/Rocket.Chat/assets/100014271/bd5292c3-45ec-4e43-9bb1-310d56e8e349)
> 

Yes, but it is an alert which notifies the user that their changes have been applied only after the button has done its work. What I am saying is when the button is submitting a form or executing a function the user should get some kind of feedback.

**Comment by saad9604** [comment]
@Sreejit-Sengupto  kindly check

**Comment by ArpitShukla3** [comment]
let me look into it
